misago/users/tests/test_djangoadmin_user.py

```diff
@@ -1,47 +1,64 @@
 from django.contrib.auth import get_user_model
 from django.contrib.auth.models import Permission
 from django.core.urlresolvers import reverse
-from django.test import override_settings
 from django.utils import formats
+from django.utils.translation import ugettext as _
+from django.test import override_settings
 
 from misago.admin.testutils import AdminTestCase
 
 
 @override_settings(ROOT_URLCONF='misago.core.testproject.urls')
 class TestDjangoAdminUserForm(AdminTestCase):
-    def test_edit_page_content(self):
-        """assert that edit-view of `test_user` contains expected content."""
-        User = get_user_model()
-
-        test_user = User.objects.create_user(
+    user = None
+    user_list_view_url = None
+    user_edit_view_url = None
```

Comment:
Yeah, in Python this makes `user` and `user_` class attributes instead of instance attributes. This may cause test state to leak from tests down the road, and is inconsistend with rest of suite, so I would remove it anyway.